docs/implementations/moby.md

```diff
@@ -0,0 +1,79 @@
+# Moby
+
+The Moby client is the open source upstream of Docker, and the workflow has been widely copied.
+
+## API Workflow
+
+- Ping the v2 endpoint (`GET /v2/`), if 401 (unauthorized status), save values from the `WWW-Authenticate` header
+- Manually set scope to `repository:${repo}:${action}` with action = `pull`, `push`, or `pull,push`
```

Comment:
what about 'push,pull'?

Reply:
It's worth double checking that last one. I know that it will make separate scopes instead of a single one with two actions if they are added separately. So the multiple action values may only be theoretical.